rest-client-reactive-quickstart/src/main/resources/application.properties

```diff
@@ -1,2 +1,4 @@
 quarkus.tls.trust-all=true
 quarkus.rest-client.extensions-api.url=https://stage.code.quarkus.io/api
+quarkus.rest-client.extensions-api.shared=true
+quarkus.rest-client.extensions-api.name=fojooh
```

Comment:
Is this normal?

Reply:
sharing is an advanced feature IMO and I don't think this should be added to the quickstart

Reply:
Left-overs from another branch.